src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -101,6 +107,6 @@ public void periodic() {
   }
 
   public void setMotor(double forwardSpeed, double turnSpeed) {
-    driveRobot.arcadeDrive(forwardSpeed, turnSpeed);
+    driveRobot.arcadeDrive(filter.calculate(forwardSpeed), turnSpeed);
```

Comment:
we're still using arcade drive? are we planning on moving towards separate control loops for left and right side?